src/index.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable  import/no-cycle, import/no-mutable-exports */
+import './mystyles.css';
+import { store } from './modules/local-storage.js';
+import { addTask } from './modules/task.js';
+import { update } from './modules/update.js';
+import { deleter } from './modules/delete.js';
+import { status, clearer } from './modules/check.js';
+import { dragStart, dragOver, drop } from './modules/dragmodule.js';
+
+const addButton = document.querySelector('.fa-arrow-right-to-bracket');
+export const todo = document.querySelector('.todo');
+export const abnormal = document.querySelector('.abnormal');
+export const domTasks = document.querySelector('.all-activities');
+const clear = document.querySelector('.clear');
+
+export let tasks = [];
```

Comment:
You've done a great job, but I want to suggest you use the 'const' keyword to declare your array. This would ensure the array is not redefined by mistake.

Reply:
Thank you so much @seyioyemade, for the time you put in to give the review. The actual reason for using let is because the variable would need to be re-assigned. Do you think there is any place in the project requirement where re-assignment is forbidden? Kindly share the requirement with me so I can refactor the code.

Reply:
Thank you so much @seyioyemade. I am closing the Pull Request thread now.